crates/astria-sequencer/src/bridge/bridge_lock_action.rs

```diff
@@ -72,24 +72,24 @@ impl ActionHandler for BridgeLockAction {
             .await
             .context("failed to get transfer base fee")?;
 
-        let transaction_hash = state
+        let transaction_id = state
             .get_current_source()
             .expect("current source should be `Some`")
-            .transaction_hash;
-        let source_transaction_index = state
-            .get_transaction_action_index()
+            .transaction_id;
+        let index_of_action = state
+            .get_transaction_index_of_action()
             .await
-            .context("failed to get transaction action index")?
-            .expect("action index should be `Some`");
+            .context("failed to get transaction index of action")?
+            .expect("index_of_action should be `Some`");
```

Comment:
I know you just adapted these panic messages, but they are kinda useless. If `index_of_action` is `None`, then you get the message "it should be `Some`". This kinda leaves it open as to why it should be set.